src/ft2font.cpp

```diff
@@ -592,8 +579,7 @@ void FT2Font::load_char(long charcode, FT_Int32 flags, FT2Font *&ft_object, bool
         ft_object = ft_object_with_glyph;
     } else {
         ft_object = this;
-        FT_UInt glyph_index = ft_get_char_index_or_warn(face, (FT_ULong)charcode);
-
+        FT_UInt glyph_index =  FT_Get_Char_Index(face, (FT_ULong) charcode);
```

Comment:
Do we wish this single instance to warn if this returns `0`?

Or are the desired warnings from this handled by the other parts of this PR?

(All other instances had `warn=false` passed, but this one would have warned)